pages/linux/sport.md

```diff
@@ -0,0 +1,32 @@
+# sport
+
+> Search and install SlackBuilds.
+> More information: <http://slackermedia.info/handbook/doku.php?id=slackbuilds>.
+
+- Pull the list of SlackBuilds to run `sport` for the first time:
+
+`sudo mkdir -p /usr/ports && sudo rsync -av rsync://slackbuilds.org /slackbuilds/$(awk '{print $2}' /etc/slackware-version)/ /usr/ports/`
+
+- Pull in any updates to the system's tree via rsync:
+
+`sudo sport rsync`
+
+- Search for a package with "substring" in its name:
+
+`sport search "{{substring}}"`
```

Comment:
Usually we go for something a bit more concise (e.g. [dnf](https://github.com/tldr-pages/tldr/blob/main/pages/linux/dnf.md), [apt](https://github.com/tldr-pages/tldr/blob/main/pages/linux/apt.md)):

```suggestion
- Search for a package by name:

`sport search "{{keyword}}"`
```